src/main/java/pl/widulinski/scrapp/webDriver/BrowserPicker.java

```diff
@@ -0,0 +1,27 @@
+package pl.widulinski.scrapp.webDriver;
+
+
+import lombok.Data;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.springframework.stereotype.Component;
+
+@Component
+@Data
+public class BrowserPicker {
+
+    private WebDriver driver;
+
+    public WebDriver setUpChromeDriver() {
+
+        System.setProperty("webdriver.chrome.driver", "C:\\chromedriver.exe");
```

Comment:
nie da się ustawić tego w `application.properties`?
nie testowałem, tak się zastanawiam czy Selenium tego nie wyłapie.
jeśli nie to i tak `C:\\chromedriver.exe` to zahardkodowana ścieżka, może warto by ją wstrzyknąć przez `application.properties` a tutaj ustawić przez `System.setProperty`

Reply:
wrzuciłem do properties